solutions/caesar_encryption.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for encrypting using caesar's cypher.
+
+Module contents:
+    - caesar_encryption: generates an encrypted string.
+
+Created on 28/12/2024
+@author: Caesar Ghazi
+"""
+
+
+def caesar_encryption(text: str, shift: int) -> str:
+    """
+    Encrypts a text using the Caesar Cipher.
+
+    Parameters:
+        text (str): The input text to encrypt.
+        shift (int): The number of positions to shift each letter.
+
+    Returns:
+        str: The encrypted text.
+
+    Raises:
+        AssertionError: if Text is not a string.
+        AssertionError: if Shift is not an integer.
+
```

Comment:
here you are missing smth like Other assumptions (eg. if it's a number, what's the expected range?)
Ensures the shift is within 0-25

Reply:
i'm sorry but i wanna make sure i get this correctly, do you mean i should make the shift is within a specific range or what? 
because right now i just made the mod 26 so it wraps around if it exceeds the alphabet range (0-25).

Reply:
yea  i know i see that but its better to right it as comment